apps/style/craft/style.scss

```diff
@@ -3,7 +3,7 @@
 $root: "/blockly/media/craft/"; //TODO: Parameterize for asset pipeline
 
 @import "color.scss";
-@import "../common";
+@import "../responsive-visualization";
```

Comment:
There was a mixin in `responsive-visualization` that we needed (which was previously being imported via `common.scss`)

Reply:
@breville curious if you recall anything from back in the dayabout Minecraft's integration with Microsoft's "Code Connection", like this project type -- that's the only thing I could think of why Minecraft might need all of common.scss:

https://studio.code.org/projects/minecraft_codebuilder/new

Reply:
Chatted thru this a bit in Slack -- this lab has been deprecated by Microsoft, so we're not too worried about styling. More detail here:

https://codedotorg.slack.com/archives/C03DBDN67B7/p1706032603054989